src/components/staff/TeacherItem.tsx

```diff
@@ -143,7 +143,7 @@ function TeacherItem({ teacher }: Props): JSX.Element {
     <TouchableOpacity
       testID="go-to-teacher"
       onPress={() => {
-        navigation.push('TeacherProfile', {
+        navigation.navigate('TeacherProfile', {
```

Comment:
If you intended for this to be `navigation.push(...)`, you should change the test, not the source code.

Here's an example that has push and navigate in the same test:
https://github.com/themeetinghouse/mobile/blob/67ea6fe7c57893cf3043cd05c05f4d3d9d5423ea/src/components/__tests__/RecentTeaching.test.js#L278-L291

Reply:
I was thinking that navigate makes more sense because push can cause the user to navigate twice. Thoughts?

Reply:
Ah I see - was just making sure you changed push to navigate for a functional reason, not just to pass the test 👍

Reply:
Thanks for checking!

Reply:
Side note: if `navigation.push(...)` is causing issues, I'd recommend refactoring the entire repo to use `navigation.navigate(...)` where appropriate.